packages/msw-addon/src/decorator.ts

```diff
@@ -21,6 +21,10 @@ const deprecateMessage = deprecate(`
 More info: https://github.com/mswjs/msw-storybook-addon/blob/main/MIGRATION.md#mswdecorator-is-deprecated-in-favor-of-mswloader
 `)
 
+/**
+ * @deprecated The `mswDecorator` is deprecated and will be removed in the next release. Please use the `mswLoader` instead.
+ * More info: https://github.com/mswjs/msw-storybook-addon/blob/main/MIGRATION.md#mswdecorator-is-deprecated-in-favor-of-mswloader
```

Comment:
Could you please link to the exact commit state instead? That way, the error always links to existing page, even if we delete that page in the future.

```suggestion
 * More info: https://github.com/mswjs/msw-storybook-addon/blob/ec35e9371f8a56a27220838fba798b9001ac7fad/MIGRATION.md#mswdecorator-is-deprecated-in-favor-of-mswloader
```

Reply:
Makes sense to me 👍